west.yml

```diff
@@ -34,13 +34,16 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: de69d2df908fc7bccd8c03b106e7645e90b1503b
+      revision: pull/116/head
     - name: fw-nrfconnect-mcuboot
       path: mcuboot
       revision: 59fde9c792bfaa36887c860fb8cd0ca1f1bc4db5
     - name: nrfxlib
       path: nrfxlib
       revision: 9bcc77b27d12162adc31d8c8c70f4e499338fbdb
+    - name: tinycbor
```

Comment:
You will need to specify the remote to be upstream here, we don't have a fork of tinycbor

Reply:
I am off today but if in doubt please ask @tejlmand

Reply:
Perhaps we should set the remote to https://github.com/zephyrproject-rtos/tinycbor? The Zephyr fork is already far behind the upstream.